client/src/views/organizations/OrganizationInformationPage.vue

```diff
@@ -3,223 +3,248 @@
+                      {{ orgInfo.users.standards }}
+                    </ion-text>
+                  </div>
+                  <!-- Outsiders if allowed -->
+                  <div
+                    v-if="orgInfo.outsidersAllowed"
+                    class="user-active-list-item"
+                  >
+                    <tag-profile :profile="UserProfile.Outsider" />
+                    <ion-text class="organization-info-value title-h4">
+                      {{ orgInfo.users.outsiders }}
+                    </ion-text>
+                  </div>
+                </div>
+              </div>
+
+              <!-- Revoked -->
+              <div class="org-user-list-item user-revoked">
+                <div class="user-revoked-header">
+                  <ion-title
+                    class="user-revoked-header__title title-h5"
+                    @click="openPopover($event)"
+                  >
+                    {{ $t('OrganizationPage.infoPage.users.revokedUsers') }}
+                  </ion-title>
+                  <span class="title-h4">{{ orgInfo.users.revoked }}</span>
+                  <!-- if we decide to show a modal with the revoked users list -->
+                  <ion-button
+                    fill="clear"
+                    size="small"
+                    class="show-users-button"
+                    v-show="false"
+                  >
+                    {{ $t('OrganizationPage.infoPage.users.seeUsers') }}
+                  </ion-button>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </template>
+      <template v-else>
+        <div class="information-not-found">
+          <ion-icon
+            :icon="warning"
+            size="large"
+          />
+          <ion-text class="body">
+            {{ $t('OrganizationPage.infoPage.getInfoFailed') }}
+          </ion-text>
+        </div>
+      </template>
     </ion-content>
   </ion-page>
 </template>
 
 <script setup lang="ts">
+import { writeTextToClipboard } from '@/common/clipboard';
 import { formatFileSize } from '@/common/file';
-import { OrganizationInfo, getOrganizationInfo } from '@/parsec';
-import { IonChip, IonContent, IonItem, IonLabel, IonList, IonListHeader, IonPage, IonTitle } from '@ionic/vue';
+import TagProfile from '@/components/users/TagProfile.vue';
+import UsersPopover from '@/components/users/UsersPopover.vue';
+import { OrganizationInfo, UserProfile, getOrganizationInfo } from '@/parsec';
+import { IonButton, IonContent, IonIcon, IonLabel, IonPage, IonText, IonTitle, popoverController } from '@ionic/vue';
+import { copy, warning } from 'ionicons/icons';
 import { Ref, onMounted, ref } from 'vue';
 
 const orgInfo: Ref<OrganizationInfo | null> = ref(null);
+const adressCopiedToClipboard = ref(false);
+
+async function copyAdress(adress: string): Promise<void> {
+  await writeTextToClipboard(adress);
```

Comment:
Errors are not handled, what if the copy fails?

Reply:
Missed it.